selfdrive/car/honda/carstate.py

```diff
@@ -126,6 +127,9 @@ def get_can_signals(CP, gearbox_msg, main_on_sig_msg):
     ]
     checks.append(("STANDSTILL", 50))
 
+  if CP.minSteerSpeed > 0.:
```

Comment:
we shouldn't use this as a gate on these new Hondas. Lots of older Hondas have > min steering speeds that we don't set properly (but we should), so we don't want to start subscribing to a random message if we set the minimum steering speed.